src/qunit.css

```diff
@@ -68,6 +68,12 @@
 	overflow: hidden;
 }
 
+#qunit-filteredTest {
+	padding: 0.5em 1em 0.5em 1em;
+	background-color: #F4FF77;
+	color: #366097;
```

Comment:
This looks like the color we also use for the "skipped" badge, which seems like a good idea, but I don't like what it looks like when it goes over the whole page. I tried some other colors but couldn't find anything that looks good. Reusing existing background colors like `#D2E0E6` works, but makes this too easy to miss.